src/clj/kixi/hecuba/api/datasets.clj

```diff
@@ -123,7 +126,17 @@
       :post (let [body (decode-body request)
                   {:keys [operation]} body]
               (if (some #{operation} ["divide" "sum" "subtract" "multiply-series-by-field"
-                                      "divide-series-by-field" "total-usage-weekly" "total-usage-monthly"])
+                                      "divide-series-by-field" "total-usage-weekly" "total-usage-monthly"
+                                      "tariff-calculation-hourly" "tariff-calculation-monthly"
```

Comment:
again, hourly is actually raw. Hourly and daily rollups are handled by the normal roll up code.